opentreemap/treemap/ecocache.py

```diff
@@ -50,12 +50,12 @@ def _get_or_compute(prefix, filter, compute_value):
 def _get_key(prefix, filter):
     if filter and (filter.filterstr or filter.displaystr):
```

Comment:
this should probably now be simplified to:
```
    if prefix == 'eco/Plot':
        version = filter.instance.eco_rev
    elif prefix == 'count/plots':
        version = filter.instance.geo_rev
    else:
        version = filter.instance.universal_rev
```
assigning to the same value in the first clause and the catch-all is confusing. Is it ok to drop out this check entirely (which we're effectively doing)?
```
    if filter and (filter.filterstr or filter.displaystr):
        version = filter.instance.universal_rev
```